### DISCUSSION TITLE
Checkbox not rendering in gist

### DISCUSSION THREAD

**Discussion by thesaravanakumar** [community/community#42876] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/59575502/209991876-193f818e-a0d1-46f1-b963-f14644122bed.png)
In the above picture, the checkbox is not rendered but when I try to create a check box in a repository it rendered properly. Is there any bug in gists?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hi @thesaravanakumar 

Just the preview that doesn't show the checkbox, but when you save it appears

**Reply by thesaravanakumar** [reply]
Oh ok, that works. thanks.
